### DISCUSSION TITLE
Does anybody think it would be better to put timestamps on messages in terminal?

### DISCUSSION THREAD

**Discussion by chunjiw** [fonsp/Pluto.jl#2968] [Pluto development]
Hi,

Most of the time when I see a message (or warning or error) in the terminal, I wonder when it is produced. I think most of us focus on the web UI when we work. So we are not looking at it when a message appears in the terminal. 

Does anyone else think it would be better to have a timestamp associated with a message in the terminal?

**Comment by pankgeorg** [comment]
This is a good idea. Docker does something similar with a flag: https://docs.docker.com/reference/cli/docker/container/logs. I'm not sure if this is better suited for implementation in _pluto_ or in Julia's logging system, but it's a nice feature, I'd definitely second it.
